libcudacxx/include/cuda/std/__cccl/compiler.h

```diff
@@ -33,7 +33,7 @@
 #  define _CCCL_MSVC_VERSION      _MSC_VER
 #  define _CCCL_MSVC_VERSION_FULL _MSC_FULL_VER
 #elif defined(__CUDACC_RTC__)
-#  define _CCCL_COMPILER_NVRTC
+#  define _CCCL_COMPILER(NVRTC) _CCCL_CUDACC_VER_MAJOR * 1000 + _CCCL_CUDACC_VER_MINOR * 10
```

Comment:
This should be `_CCCL_COMPILER_NVRTC`, not `_CCCL_COMPILER(NVRTC)`.

Plus the compiler version cannot be defined like that, because `_CCCL_COMPILER` computes the version using the `_CCCL_COMPILER_MAKE_VERSION` macro, where the standard `100 * major + minor` will be used. We either have to change the design and write the "make version" macros for each compier, or simply change `100` to `1000` in the `_CCCL_COMPILER_MAKE_VERSION` macro, it will work fine.

Reply:
Yeah thanks a lot, that was a mistake from the search an replace.

I moved to the calculated version because we never ever use the nvrtc version because it is just assumed to be the nvcc version